source/documentation/deploying_apps/index.erb

```diff
@@ -159,6 +159,14 @@ You must create a [network policy](https://cli.cloudfoundry.org/en-US/cf/add-net
 cf add-network-policy PUBLIC_APPNAME --destination-app PRIVATE_APPNAME --protocol tcp --port 8080
 ```
 
+#### Recreate the network policy (optional)
+
+If you are using [version 2.0 of the Cloud Foundry API](https://github.com/cloudfoundry/cloud_controller_ng/tree/master/docs/v2) and a [blue-green deployment tool](https://docs.cloudfoundry.org/devguide/deploy-apps/blue-green.html).
```

Comment:
This feels quite fragmented. I think it would clearer to repeat "recreate the network policy" after the big "If" here. But I'm not a professional writer :)

Reply:
I made a mistake here and missed off the end of the sentence. My mistake and thank you for spotting that.